packages/docusaurus-theme-translations/locales/tk/theme-common.json

```diff
@@ -0,0 +1,74 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "Ýapmak",
+  "theme.BackToTopButton.buttonAriaLabel": "Başyna süýşmek",
+  "theme.CodeBlock.copied": "Göçürildi",
+  "theme.CodeBlock.copy": "Göçürmek",
+  "theme.CodeBlock.copyButtonAriaLabel": "Bufere göçürmek",
+  "theme.CodeBlock.wordWrapToggle": "Setir boýy geçir",
+  "theme.DocSidebarItem.collapseCategoryAriaLabel": "Gapdal paneli gizle '{label}'",
+  "theme.DocSidebarItem.expandCategoryAriaLabel": "Gapdal paneli görkez'{label}'",
+  "theme.ErrorPageContent.title": "Sahypada näsazlyk ýüze çykdy.",
+  "theme.ErrorPageContent.tryAgain": "Gaýtadan synanyşyň",
+  "theme.NavBar.navAriaLabel": "Esasy",
+  "theme.NotFound.p1": "Gynansakda, ýüzlenen sahypaňyz tapylmady",
+  "theme.NotFound.p2": "Web sahypanyň dolandyryjylaryna habar bermegiňizi sizden haýyş edýäris",
+  "theme.NotFound.title": "Sahypa tapylmady",
+  "theme.TOCCollapsible.toggleButtonLabel": "Bu sahypanyň mazmuny",
+  "theme.admonition.caution": "duýduryş",
+  "theme.admonition.danger": "howp",
+  "theme.admonition.info": "maglumat",
+  "theme.admonition.note": "bellik",
+  "theme.admonition.tip": "maslahat",
+  "theme.admonition.warning": "duýduryş",
+  "theme.blog.archive.description": "Arhiw",
+  "theme.blog.archive.title": "Arhiw",
+  "theme.blog.paginator.navAriaLabel": "Blog sahypasynda nawigasiýa",
+  "theme.blog.paginator.newerEntries": "Täze ýazgylar",
+  "theme.blog.paginator.olderEntries": "Köne ýazgylar",
+  "theme.blog.post.paginator.navAriaLabel": "Blog ýazgysynyň sahypasyndaky nawigasiýa",
+  "theme.blog.post.paginator.newerPost": "Täze ýazgy",
+  "theme.blog.post.paginator.olderPost": "Köne ýazgy",
+  "theme.blog.post.plurals": "{count} ýazgy|{count} ýazgylar|{count} ýazgylary",
```

Comment:
CLDR says Turkmen only has 2 cardinal forms

https://www.unicode.org/cldr/cldr-aux/charts/34/supplemental/language_plural_rules.html#:~:text=Turkmen,tk

My browser also says so:

```js
function getLocalePluralForms(locale) {
  const AllPluralForms = ['zero','one','two','few','many','other']
  const pluralCategories = new Intl.PluralRules(locale).resolvedOptions().pluralCategories;
  pluralCategories.sort((c1,c2) => AllPluralForms.indexOf(c1) > AllPluralForms.indexOf(c2) ? 1 : -1);
  return pluralCategories;
}

const myLocale = "tr"; // Change this variable!
console.log("Plural forms for this locale are =>>> ",getLocalePluralForms(myLocale)); 
```

So, why are you using 3 plural forms?

Reply:
Here as you can mention plural changes its form. So 3rd form is important in this particular case. 

Regarding the code snippet you provided, I have executed it before I made PR, but as I mentioned before, the plural form changes itself according to the accusative form of speech (in formal written Turkmen language, it is crucially important). That is the reason why I made it to 3 forms everywhere in the translations

Reply:
I don't know Turkmen, but the problem is that if you provide 3 forms, the 3rd one will never ever be used by our translation system, so it's better to provide 2. 

I don't think our system atm is able to handle "accusative form of speech" so please only consider the 2 CLDR "one"+"other" plural forms. It seems to be a particularity of your language that the JS Intl API is maybe not designed to solve. It is unlikely we will be able to support this in Docusaurus unless JS add support for it first 😅 .

I can't advise more what you should do exactly, but please only keep 2 plural forms. Otherwise I'll merge by just removing the last one (because it won't be used anyway so it won't change anything in terms of outcomes)

Reply:
Ok ok, I got it, I will update it soon